sdks/python/apache_beam/runners/common.py

```diff
@@ -558,16 +568,33 @@ def __init__(self, window_fn, main_receivers, tagged_receivers):
     self.main_receivers = main_receivers
     self.tagged_receivers = tagged_receivers
 
+    # TODO: remove if block after per-element-output-counter released
+    # TODO: [BEAM-3937]
+    experiments = RuntimeValueProvider.get_value('experiments', str, [])
+    if 'outputs_per_element_counter' in experiments:
+      self.has_per_element_output_count = True
```

Comment:
You can also avoid the `has_per_element_output_count` property by just checking `if self.per_element_output_counter: ...`

Reply:
I assumed a boolean variable may be more readable. But I'll get ride of this in next revision.